.travis.yml

```diff
@@ -38,6 +52,20 @@ matrix:
       compiler: clang
       env: [CONFIGURATION=Release, *gcc-warnings]
 
+  # Sporadic failure builds
+    - compiler: gcc
+      env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+    - compiler: gcc
+      env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+
+  allow_failures:
+    - env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+    - env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+
```

Comment:
The per-job is exactly a path I was searching for. This makes the most sense, but it doesn't look like it exists right now.

What about pre-processor expansion? For example, is there such a syntax where we could declare `STRESSTEST="CONFIGURATION=Debug    CTEST_OPTS=\"--repeat-until-fail 25 --schedule-random\""` at the top of the YML definition, and then define lines 57, 61, 66, and 67 above as `$STRESSTEST`? (I'm not such a pre-processor or expansion exists; just thinnking aloud)